packages/ingest/src/granule.ts

```diff
@@ -296,48 +381,7 @@ async function moveGranuleFiles(sourceFiles, destinations) {
 
   await Promise.all(moveFileRequests);
 
-  return processedFiles;
-}
-
-/**
-  * rename s3 file with timestamp
-  *
-  * @param {string} bucket - bucket of the file
-  * @param {string} key - s3 key of the file
-  * @returns {Promise} promise that resolves when file is renamed
-  */
-async function renameS3FileWithTimestamp(bucket, key) {
```

Comment:
Do we need to flag this deletion since it was exported? And the function below?

Reply:
This function is still there and exported, it just had to be moved higher in the file so that it was declared before it was used. I'll update the Changelog to note that the `@cumulus/ingest/granule.getRenamedS3File` function has been renamed to
  `listVersionedObjects`.